reference-architectures/cloud_deploy_flow/CloudBuild/buildCloudRun.yaml

```diff
@@ -0,0 +1,27 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker:20.10:24'
+  dir: 'CloudRun/'
+  args: ['build',
+   '--tag',
+   'us-central1-docker.pkg.dev/$PROJECT_ID/random-date-app/app:latest',
+   '.'
+   ]
+- name: 'gcr.io/cloud-builders/docker:20.10:24'
+  dir: 'CloudRun/'
+  args: ['push',
+  'us-central1-docker.pkg.dev/$PROJECT_ID/random-date-app/app:latest'
+  ]
+- name: 'gcr.io/cloud-builders/gcloud:latest'
```

Comment:
To increase build reproducibility, consider using fixed container image tags, or even hashes